src/envo/envo-edit.owl

```diff
@@ -35086,7 +35081,7 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_01001777> "multiplet earthquake")
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Doublet_earthquake>) rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001777> "The mainshocks may be separated by seconds or years. Mainshocks must be within 1.2 units of magnitude of one another and have nearly identical seismic waveforms.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001777> "multiplet tectonic earthquake"@en)
-EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_01001777> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01000678> ObjectMinCardinality(2 <http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01001771>)))
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_01001777> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01000678> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01001771>)))
```

Comment:
This one isn't right. Recall that it is always sound to replace the RHS of a SubClassOf with a more general class expression, but this shouldn't be done with an EquivalenceAxiom.

Here is the axiom:

'multiplet tectonic earthquake' EquivalentTo 'tectonic earthquake' and ('has part' min 2 mainshock)

This should be changed to:

'multiplet tectonic earthquake' SubClassOf 'tectonic earthquake' and ('has part' some mainshock)

once this is done the PR can be merged

Note that even if mixing cardinality and transitivity were permitted in DL, this logical definition would not be correct; the text def is _A tectonic earthquake which has two or more mainshocks of similar magnitude, and which originate from the same fracture zone and stress field._. 

The text def should always match the logical def (see the SRS guidelines); here the tex def imposes conditions that are not reflected in the logical def so a weaker axiom is called for